ydb/docs/redirects.yaml

```diff
@@ -292,6 +292,15 @@ common:
   - from: /reference/ydb-cli/export_import/tools_restore.md
     to: /reference/ydb-cli/export-import/tools-restore.md
 
+
+# YQL syntax redirects
+  - from: /yql/reference/syntax/alter_table.md
+    to: /yql/reference/syntax/alter_table/index.md
+  - from: /yql/reference/syntax/select.md
+    to: /yql/reference/syntax/select/index.md
+  - from: /yql/reference/syntax/create_table.md
+    to: /yql/reference/syntax/create_table/index.md    
```

Comment:
This is probably not necessary as `foo.md` and `foo/index.md` leads to the same URL.

Reply:
It is even harmful:
![Screenshot 2024-06-14 at 11 25 33](https://github.com/ydb-platform/ydb/assets/623861/932668db-ccac-4109-8c0f-fc333c058781)